pkg/config/cli_test.go

```diff
@@ -0,0 +1,59 @@
+/*
+Copyright 2022 k0s authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package config
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+	"golang.org/x/exp/slices"
+)
+
+func TestAvailableComponents_SortedAndUnique(t *testing.T) {
+	t.Parallel()
+
+	expected := slices.Clone(availableComponents)
+	slices.Sort(expected)
+
+	assert.Equal(t, expected, availableComponents, "Available components aren't sorted")
+
+	slices.Compact(expected)
+	assert.Equal(t, expected, availableComponents, "Available components contain duplicates")
+}
+
+func TestControllerOptions_Normalize(t *testing.T) {
```

Comment:
Since the other test is run with `t.Parallel()`, perhaps this one should be also ran with `t.Parallel()` as well.
Other than that, LGTM.

Reply:
Doh. Will remove the other Parallel as well to make it homogeneous again. Technically those tests could be parallelized, but since t.Parallel is sometimes tricky (depending on the test setup), it might make sense to not use it at all.

Reply:
Fair enough